sql/core/src/test/scala/org/apache/spark/sql/execution/WholeStageCodegenSuite.scala

```diff
@@ -149,4 +149,75 @@ class WholeStageCodegenSuite extends SparkPlanTest with SharedSQLContext {
       assert(df.collect() === Array(Row(1), Row(2)))
     }
   }
+
+  test("SPARK-21603 check there is a too long generated function") {
+    val ds = spark.range(10)
```

Comment:
Ok, I have modified it as you suggested above all, would you like to review it again, thanks.